tests/blobcache.bats

```diff
@@ -55,6 +55,37 @@ function _check_matches() {
         expect_output --from="$unmatched" "$5"  "$6 should not match"
 }
 
+# Integration test for https://github.com/containers/image/pull/1645
+@test "blobcache: blobs must be reused when pushing across registry" {
+	start_registry
+	run_buildah login --tls-verify=false --authfile ${TEST_SCRATCH_DIR}/test.auth --username testuser --password testpassword localhost:${REGISTRY_PORT}
+	run_buildah build $WITH_POLICY_JSON -t exp -f $BUDFILES/simple-multi-step/Containerfile
+	run_buildah push $WITH_POLICY_JSON --format docker exp dir:${TEST_SCRATCH_DIR}/exp
```

Comment:
To confirm I am reading this right, this creates a _compressed_ representation, doesn’t it? (And that is fine and required, otherwise, after clearing the BIC, there would be no knowledge of a compressed digest.)

Reply:
Yes this creates a compressed version.